app/uk/gov/hmrc/gform/models/helpers/TaxPeriodHelper.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.models.helpers
+import uk.gov.hmrc.gform.validation.{ ComponentField, FormFieldValidationResult }
+import uk.gov.hmrc.gform.views.summary.TextFormatter
+import uk.gov.hmrc.gform.models.helpers.DateHelperFunctions.{ getMonthValue, renderMonth }
+
+object TaxPeriodHelper {
+
+  def formatTaxPeriodOutput(valResult: Option[FormFieldValidationResult]) = {
+    val mapOfResults = valResult.get match { case ComponentField(a, b) => b }
+    mapOfResults.keySet.toString.split('(')(2).split(',')(0)
```

Comment:
This has a bad smell!  But @josef-vlach tells me it is not used and is replaced by something better

Reply:
yes this is a quick solution I came up with but we replaced it. I was just a case of forgetting to delete it before committing.